pkg/workload/mysql/testingtool.go

```diff
@@ -74,6 +74,15 @@ func (t *TestingTool) Run(kubeClient kubernetes.Interface, testingCase workload.
 	if err != nil {
 		return errors.Errorf("helm install failed, ret:%s, error:%v", strings.Join(ret, "\n"), err)
 	}
+	// Delete the release of the workload
+	defer func() {
+		ret, delErr := util.RunCommand("helm", "del", "--purge", t.Workload.Helm.Name)
+		if delErr != nil && err == nil {
+			glog.Warningf("helm delete failed,ret:%s,error:%v", strings.Join(ret, "\n"), delErr)
+			err = delErr
+			return
+		}
+	}()
```

Comment:
we should delete the release in `mysql.go` file when finish

Reply:
ok